api/src/config.rs

```diff
@@ -29,6 +29,8 @@ pub struct ConfigV2 {
     pub cache: Option<CacheConfigV2>,
     /// Configuration information for RAFS filesystem.
     pub rafs: Option<RafsConfigV2>,
+    /// Configuration information for image deduplication.
+    pub deduplication: Option<DeduplicationConfigV2>,
```

Comment:
Please add [smoke test](https://github.com/dragonflyoss/nydus/tree/master/smoke/tests) and update [doc](https://github.com/dragonflyoss/nydus/tree/master/docs)